manifests/windows/alias.pp

```diff
@@ -0,0 +1,28 @@
+class classroom::windows::alias {
+
+  assert_private('This class should not be called directly')
+
+  $ps_module_path = 'C:/Users/Administrator/Documents/WindowsPowerShell/Modules'
+  $alias_dir      = "${ps_module_path}/alias"
+  $psm_file       = 'alias.psm1'
+  $psd_file       = 'alias.psd1'
```

Comment:
Correct. Created, then New-ModuleManifest command is run, then module is imported. Perhaps there is a simpler way to accomplish this that I'm not familiar with